fluid/src/common/type_define.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2016 Baidu, Inc. All Rights Reserved.
```

Comment:
Do not copy-n-paste code!

cmake supports sub-projects. You can use `git submodule` and make paddle as a submodule of paddle-mobile. Use cmake sub-projects to import paddle as a subproject.

Reply:
CMakeLists.txt in fluid folder has include all the .h and .cpp file, so why we need a submodule in the common folder ?